docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,212 @@
+for search on all indices.
+|`type` |String |Column field header type.
+|==============================================
+
+[[dataProvider-obj]]
+[discrete]
+==== dataProviders object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`and` |dataProviders[] |Array containing statements using `AND` logic.
+|`id` |String |The dropzone query statement's unique ID.
+|`name` |String |The dropzone query statement's name (the statement's value
+when timelines are exported).
+|`enabled` |Boolean |Indicates if the dropzone query statement is enabled.
+|`excluded` |Boolean |Indicates if the dropzone query statement uses `NOT`
+logic.
+|`kqlQuery` |String |Always empty.
+|`queryMatch` |QueryMatchInput a|The dropzone query statement:
+
+* `displayField`: ???
+* `displayValue`: ???
+* `field`: The field used to filter results.
+* `operator`: The statements operator.
+* `value`: The field value used to match results.
+
+|==============================================
+
+[[eventNotes-obj]]
+[discrete]
+==== eventNotes object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`created` |Float |The time the note was created, using a 13-digit Epoch
+timestamp.
+|`createdBy` |String |The user who added the note.
+|`eventId` |String |The ID of the event to which the note was added.
+|`note` |String |The note's text.
+|`noteId` |String |The note's ID
+|`timelineId` |String |The ID of the timeline to which the note was added.
+|`updated` |Float |The last time the note was updated, using a
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the note.
+|`version` |String |The note's version.
+|==============================================
+
+[[favorite-obj]]
+[discrete]
+==== favorite object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`favoriteDate` |Float |The time the timeline was marked as a favorite.
+|`fullName` |String |The full name of the user who marked the timeline as
+a favorite.
+|`keySearch` |String |???
+|`userName` |String |The {kib} username of the user who marked the
+timeline as a favorite.
+|==============================================
+
+[[filters-obj]]
+[discrete]
+==== filters object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`exists` |String |Field exists query, or `null` when not defined.
+|`meta` |FilterMetaTimelineInput a|Object containing filter details:
+
+* `alias`: UI filter name.
+* `disabled`: Indicates if the filter is disabled.
+* `field`: ???
+* `key`: Field used for filtering results.
+* `negate`: Indicates if the filter query statement uses `NOT` logic.
+* `params`: Query values ???
+* `type`: Type of filter.
+* `value`: 
+
+|`match_all` |String |???
+|`missing` |String |???
```

Comment:
It’s not necessary to document these fields because they are an implementation detail of the saved filters, which can be re-used throughout Kibana. If that format is documented somewhere else, perhaps we can point to it, but otherwise it’s not necessary to document it.